### PR TITLE
Pbi 37 38 39 40 submission

### PR DESCRIPTION
**Pull request by nofamex**

### CODE REVIEW

File: src/Controllers/SubmissionController.php [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+            }
+
+            $submission = Submission::create([
+                'assignment_id' => $request->input('assignment_id'),
+                'user_id' => $user->id,
+                'file_url' => $request->input('file_url'),
+                'link_url' => $request->input('link_url'),
+            ]);
+
+            return ApiRespOnse::success($submission->toArray());
```
Comment (AldiNFitrah) [inline src/Controllers/SubmissionController.php]: I think it should be `ApiResponse`

File: src/Controllers/SubmissionController.php [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+                throw ValidationException::withMessages([
+                    'id' => 'Must enroll the course to submit assignment',
+                ]);
+            }
+
+            $response['file_url'] = $submission->file_url;
+            $response['link_url'] = $submission->link_url;
+        }
+
+        return ApiRespOnse::success($response);
```
Comment (AldiNFitrah) [inline src/Controllers/SubmissionController.php]: I think it should be `ApiResponse`

File: src/Controllers/SubmissionController.php [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+                    'date' => 'You have passed submission due date',
+                ]);
+            }
+
+            $submission->fill($request->only([
+                'file_url',
+                'link_url',
+            ]))->save();
+
+            return ApiRespOnse::success($submission->toArray());
```
Comment (AldiNFitrah) [inline src/Controllers/SubmissionController.php]: I think it should be `ApiResponse`

**Review by AldiNFitrah — APPROVED** [review]
Thats it, please do merge if you already got 2 approvals. Thank you!

**Review by kanziebub — APPROVED** [review]
great job uwu

**Review by IlmaMannix — APPROVED** [review]
gooOod job opall

**Pull request merged**